docs/setting-up-your-environment.md

```diff
@@ -7,35 +7,45 @@ In order to start engaging with the Gro API Client, you will need to set up your
 
 ## System Prerequisites
 
-The Gro API Client requires the following OS-dependant system requirements.
+The Gro API Client requires the following OS-dependent system requirements.
 
-MacOS and Linux
+### MacOS and Linux
 
-1. git (Installation instructions)
-2. python (2.7.x or 3.x) (2 Installation instructions / 3 Installation instructions)
-3. pip (Installation instructions. Note "pip is already installed if you are using Python 2 >=2.7.9 or Python 3 >=3.4")
+1. git ([Installation instructions](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git))
+2. python version 3.5 or above from <python.org>. Support for Python 2.7.13 or above is also maintained, but with its [End of Life](https://mail.python.org/pipermail/python-dev/2018-March/152348.html) soon approaching, Python 3 is recommended.
+3. pip ([Installation instructions](https://pip.pypa.io/en/stable/installing/). Note "pip is already installed if you are using Python 2 >=2.7.9 or Python 3 >=3.4")

```

Comment:
Can remove pip

I removed it in https://github.com/gro-intelligence/api-client/pull/94/files#diff-f59a9812b30c92b2866ad64e35a41ac9L5 when our required Python versions were increased, since all of our supported versions now include pip.

Reply:
Done